### PR TITLE
Fix #70 name error

### LINKED ISSUE #70: NameError: name 'trips' is not defined
**Describe the bug**
When starting the server (option `-r`) with a fresh install, the program raises an exception in `views.py`:
```
2021-04-04 20:33:14,868 :: ERROR :: update_trips: Traceback (most recent call last):
  File "/Users/udo/Develop/PSACarController/web/views.py", line 142, in update_trips
    trips = next(iter(trips_by_vin.values()))  # todo handle multiple car
StopIteration

Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.9/3.9.2_4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/local/Cellar/python@3.9/3.9.2_4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/udo/Develop/PSACarController/web/app.py", line 42, in start_app
    import web.views
  File "/Users/udo/Develop/PSACarController/web/views.py", line 204, in <module>
    update_trips()
  File "/Users/udo/Develop/PSACarController/web/views.py", line 149, in update_trips
    min_date = trips[0].start_at
NameError: name 'trips' is not defined
```

**To Reproduce**
Start program with `python3 server.py -r` with a fresh install, after decoding MyOpel app (V 1.28.2). Problem persists after restart.

**Environment:**
 - OS: MacOS 
 - Android app Version MyOpel 1.28.2 
 - Opel Corsa-e **(without onboard navigation unit)**
 - Application version: master, commit 84c798385f1495ae57f64d4d94f8db7c98578322

**Additional context**
I will try to provide a corresponding fix pull request soon

### PR DESCRIPTION
**Pull request by schombat**
Fixes #70 by handling case with empty trips and defining defaults.

### CODE REVIEW

File: web/figures.py [hunk 1]
```diff
@@ -93,21 +94,28 @@ def get_figures(trips: Trips, charging: Tuple[dict]):
         data=[tr.get_info() for tr in trips[::-1]],
         page_size=50
     )
-    # consumption_fig
-    consumption_df = DataFrame.from_records(trips.get_long_trips())
-    consumption_fig = px.line(consumption_df, x="date", y="consumption", title='Consumption of the car')
-    consumption_fig.update_layout(yaxis_title="Consumption kWh/100Km")
+    # defaults for empty trips, based on official Corsa-e c
+    kw_per_km = 17.4
```
Comment (xammmue) [inline web/figures.py:98]: shouldn't the default consumption be based on the actual vehicle instead of just one model?

File: web/views.py [hunk 2]
```diff
@@ -137,14 +137,24 @@ def after_request(response):
 def update_trips():
     global trips, chargings, cached_layout
     logger.info("update_data")
+    trips = Trips()
+    #chargings = tuple()
```
Comment (xammmue) [inline web/views.py:141]: can this commented code be removed?

File: web/views.py [hunk 3]
```diff
@@ -137,14 +137,24 @@ def after_request(response):
...
     try:
+        chargings = MyPSACC.get_chargings()
         trips_by_vin = Trips.get_trips(myp.vehicles_list)
         trips = next(iter(trips_by_vin.values()))  # todo handle multiple car
-        chargings = MyPSACC.get_chargings()
+#        chargings = MyPSACC.get_chargings()
```
Comment (xammmue) [inline web/views.py:146]: can probably also be removed now

**Comment by flobz** [comment]
Hello,
Thanks for you contribution but there is to much change in one commit. I did my own fix f7353e3.  
I didn't understand all change you made.

**Pull request closed without merging**